miso-dto/pom.xml

```diff
@@ -22,7 +22,7 @@
     </dependency>
     <dependency>
       <groupId>uk.ac.bbsrc.tgac.miso</groupId>
-      <artifactId>miso-service</artifactId>
+      <artifactId>sqlstore</artifactId>
```

Comment:
Is this really necessary? The logical flow of these as separable modules is really falling apart.